content/tutorial-static_equilibrium.md

```diff
@@ -39,8 +39,9 @@ import matplotlib.pyplot as plt
 
 In this tutorial you will use the following NumPy tools:
 
-* [`np.linalg.norm`](https://numpy.org/doc/stable/reference/generated/numpy.linalg.norm.html) : this function determines the measure of vector magnitude
-* [`np.cross`](https://numpy.org/doc/stable/reference/generated/numpy.cross.html) : this function takes two matrices and produces the cross product
+* [`np.linalg.norm`](https://numpy.org/doc/stable/reference/generated/numpy.linalg.norm.html) : This function determines the measure of vector magnitude
```

Comment:
@rossbar - I wonder whether we can instersphinx these API docs links?

Reply:
Yeah the current policy is to use explicit links to ensure that the links render properly in the html *and* in the notebooks themselves.

We should revisit this at some point, as it is a knock on maintainability. We should also check on the status of MystJS to see whether/how much of the myst language spec is supported in browser - that would be a strong motivation for the switch IMO!

Reply:
Myst got a new developer/maintainer, to we can tag him if something turns out to be upstream (I already cleared it with him). I'll try to remember to open this topic as an issue.